dataactcore/migrations/versions/5576159ed137_add_cfda_title_to_published.py

```diff
@@ -0,0 +1,37 @@
+"""Add cfda title to published
+
+Revision ID: 5576159ed137
+Revises: f15485f0092b
+Create Date: 2017-05-14 21:30:24.007049
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5576159ed137'
+down_revision = 'f15485f0092b'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_data_broker():
+    op.add_column('detached_award_financial_assistance', sa.Column('cfda_title', sa.Text(), nullable=True))
```

Comment:
It shouldn't be adding to the detached one, we aren't asking them to include the title in their file, we're deriving it on submit